Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/GameServerSocnetController.java

```diff
@@ -7,17 +7,27 @@
 import java.io.IOException;
 
 /**
+ * Controller for the socket connections from the GameServer.
+ *
  * @author PTS34A
  */
 public final class GameServerSocnetController {
 
     private final int port;
     private Server server;
 
+    /**
+     * Initializes the gamServerSocnetController class.
+     *
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)